src/Components/WebAssembly/Build/testassets/standalone/Program.cs

```diff
@@ -6,6 +6,8 @@ public class Program
     {
         public static void Main(string[] args)
         {
+            GC.KeepAlive(typeof(RazorClassLibrary.RCLTest));
```

Comment:
Why was this change made here?

Reply:
So that the linker doesn't strip the reference out very likely.

Reply:
Yup, basically what @javiercn said. The linker removes assemblies that do not appear in the build output.